python_modules/dagit/dagit/webserver.py

```diff
@@ -124,6 +123,15 @@ async def download_debug_file_endpoint(self, request: Request):
         return StreamingResponse(result, media_type="application/gzip")
 
     async def download_notebook(self, request: Request):
+        try:
+            import nbconvert
+        except ImportError:
+            return HTMLResponse(
+                "Notebook support requires nbconvert, which is not installed. You can install nbconvert using Dagit's 'notebook' extra via <code>pip install dagit[notebook]</code>"
+            )
+
+        from nbconvert import HTMLExporter
```

Comment:
should probably just do the useful import in the try block instead of this - i feel like this double import set-up is more fragile